src/app/auth/login/login.component.spec.ts

```diff
@@ -67,7 +66,7 @@ describe('LoginComponent', () => {
     it('Login: Datos no enviados por password vacía', () => {
       let email = component.loginForm.controls['email']
       let password = component.loginForm.controls['password']
-      email.setValue('jescoevas@hotmail.com')
+      email.setValue('seperu.7@gmail.com')
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/ComfyStream/ComfyStream-Frontend/pullRequest?prid=7138940)
```suggestion
      email.setValue("seperu.7@gmail.com")
```